web/src/frontend/src/digisos/skjema/ettersendelse/ettersendelse.tsx

```diff
@@ -1,225 +1,230 @@
+        this.props.dispatch(opprettEttersendelse(brukerbehandlingskjedeId));
+        this.props.dispatch(lesEttersendelser(brukerbehandlingskjedeId));
+    }
+
+    manglendeVedleggDato() {
+        const {originalSoknad, ettersendelser} = this.props;
+        let datoManglendeVedlegg: string = "";
+        if (originalSoknad) {
+            datoManglendeVedlegg = originalSoknad.innsendtDato;
+        }
+        if (ettersendelser && ettersendelser.length > 0) {
+            datoManglendeVedlegg = ettersendelser[ettersendelser.length - 1].innsendtDato;
+        }
+        return datoManglendeVedlegg;
+    }
+
+    antallManglendeVedlegg() {
+        return this.props.manglendeVedlegg.filter((item: any) => {
+            return !(item.type === "annet|annet");
+        }).length;
+    }
+
+    isEttersendelseAktivert() {
+        return this.props.originalSoknad != null && this.props.originalSoknad.orgnummer != null;
+    }
+
+    render() {
+        const {originalSoknad, ettersendelser} = this.props;
+        const antallManglendeVedlegg = this.antallManglendeVedlegg();
+        const datoManglendeVedlegg = this.manglendeVedleggDato();
+        const ettersendelseAktivert = this.isEttersendelseAktivert();
+
+        const opprettNyEttersendelseFeilet: boolean =
+            this.props.feilKode === EttersendelseFeilkode.NY_ETTERSENDELSE_FEILET;
+
+        return (
+            <div className="ettersendelse">
+                <BannerEttersendelse>
+                    <FormattedMessage id="applikasjon.sidetittel"/>
+                </BannerEttersendelse>
+                <div className="blokk-center panel ettersendelse__panel">
+                    <p className="ettersendelse ingress">
+                        <FormattedHTMLMessage id="ettersendelse.ingress"/>
+                    </p>
+
+                    {originalSoknad && (
+                        <AvsnittMedMarger
+                            venstreIkon={MargIkoner.OK}
+                            hoyreIkon={MargIkoner.PRINTER}
+                            onClickHoyreIkon={() => this.skrivUt()}
+                        >
+                            <h3><FormattedHTMLMessage
+                                id="ettersendelse.soknad_sendt"/> {originalSoknad.navenhet} kommune</h3>
+                            <p>Innsendt {originalSoknad.innsendtDato} kl. {originalSoknad.innsendtTidspunkt}</p>
+                        </AvsnittMedMarger>
+                    )}
+
+                    {ettersendelser && ettersendelser.length > 0 && ettersendelser.map((ettersendelse: any) => {
+                            return (
+                                <AvsnittMedMarger
+                                    venstreIkon={MargIkoner.OK}
+                                    key={ettersendelse.behandlingsId}
+                                >
+                                    <h3>
+                                        {ettersendelse.innsendteVedlegg.length} &nbsp;
```

Comment:
&nbsp?

Reply:
Det er for å få formateringa riktig. Legger til mellomrom :)